### PR TITLE
COMMERCE-1299 Use CommerceAccountId instead of UserId for addresses

### PR DESCRIPTION
**Pull request by alecsloan**
Let me know if you have any questions.

### CODE REVIEW

**Comment by ethanbustad** [comment]
ci:test

File: commerce-address-content-web/src/main/java/com/liferay/commerce/address/content/web/internal/display/context/CommerceAddressDisplayContext.java [hunk 1]
```diff
@@ -94,6 +94,15 @@ public String getAddCommerceAddressURL() {
 		return portletURL.toString();
 	}
 
+	public CommerceAccount getCommerceAccount() throws PortalException {
+		ThemeDisplay themeDisplay =
+			(ThemeDisplay)_httpServletRequest.getAttribute(
+				WebKeys.THEME_DISPLAY);
+
+		return CommerceAccountLocalServiceUtil.getPersonalCommerceAccount(
```
Comment (marco-leo) [inline commerce-address-content-web/src/main/java/com/liferay/commerce/address/content/web/internal/display/context/CommerceAddressDisplayContext.java]: @alecsloan can we swich this to use current account from account helper ?

**Review by marco-leo — CHANGES_REQUESTED** [review]

File: commerce-address-content-web/src/main/java/com/liferay/commerce/address/content/web/internal/display/context/CommerceAddressDisplayContext.java (hunk 1 above)
Comment (alecsloan) [reply commerce-address-content-web/src/main/java/com/liferay/commerce/address/content/web/internal/display/context/CommerceAddressDisplayContext.java]: See new commit

**Comment by alecsloan** [comment]
ci:test

**Comment by ethanbustad** [comment]
Errors should be fixed by https://github.com/liferay/com-liferay-commerce/pull/838 and https://github.com/liferay/com-liferay-commerce/pull/841.

**Comment by ethanbustad** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:stop

**Comment by ethanbustad** [comment]
ci:test
didn't update the private repo with the latest fixes :(

**Pull request merged**